### PR TITLE
LPS-202133 - Add missing action properties

### PR DESCRIPTION
**Pull request by juanjofgliferay**
Story / Bug: [LPS-191489](https://liferay.atlassian.net/browse/LPS-191489) / [LPS-201133](https://liferay.atlassian.net/browse/LPS-202133)

## Issue
Admin user can define async actions for dataset items. Some properties of the async action were not set properly, like `method` or `errorMessage` and `successMessage`.

## Solution
Set `DELETE` as default method when selecting _async_ action type.
Send `method`, `errorMessage` and `successMessage` to the dataset.

## UI
[Screencast from 2023-12-05 08-56-02.webm](https://github.com/liferay-frontend/liferay-portal/assets/132653342/ee702c7b-99d3-4a9a-bb1d-577e04dba760)

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx [hunk 1]
```diff
@@ -550,6 +550,11 @@ const ActionForm = ({
 									onChange={(event) =>
 										setActionData({
 											...actionData,
+											method:
+												event.target.value ===
+												ACTION_TYPE.ASYNC
+													? ACTION_METHOD.DELETE
+													: '',
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx:557]: I'm ok with being opinionated with default, but it seems unnecessarily sneaky to add it in handler. Could we instead set it in UI? Something like
```js
value={actionData.method || ACTION_METHOD.DELETE}
```
on [this line](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L592). The field is required, so it cannot be unset by user, so it should end up applied [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L227-L229).

**Review by markocikos — COMMENTED** [review]
@juanjofgliferay please see comment inline

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx (hunk 1 above)
Comment (juanjofgliferay) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx:557]: Yep, much better :+1:

**Comment by juanjofgliferay** [comment]
ci:forward

**Comment by juanjofgliferay** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:forward:force

**Pull request closed without merging**